CHANGELOG.rst

```diff
@@ -26,6 +26,7 @@ Changelog
   and :class:`~cryptography.hazmat.primitives.ciphers.algorithms.ARC4` into
   :doc:`/hazmat/decrepit/index` and deprecated them in the ``cipher`` module.
   They will be removed from the ``cipher`` module in 48.0.0.
+* Added support for deterministic ECDSA (:rfc:`6979`)
```

Comment:
Please add a link to the relevant code -- making ECDSA a link to the class should be enough.

Reply:
fixed